src/nni_manager/rest_server/restValidationSchemas.ts

```diff
@@ -31,10 +31,14 @@ export namespace ValidationSchemas {
                 passwd: joi.string(),
                 sshKeyPath: joi.string(),
                 passphrase: joi.string(),
-                gpuIndices: joi.string()
+                gpuIndices: joi.string(),
+                maxTrialNumPerGPU: joi.number(),
+                useActiveGPU: joi.boolean()
```

Comment:
For gpuNum and gpuIndices, 'gpu' is lower case,  suggest to use 'maxTrialNumPerGpu', 'useActiveGpu' for consistency.
And better to add document for these 2 new configuration like gpuNum and gpuIndices.

Reply:
agree

Reply:
set to `maxTrialNumPerGpu` and `useActiveGpu`